app/views/lectures/_new.html.erb

```diff
@@ -52,23 +52,34 @@
              id="new-lecture-term-error">
         </div>
       </div>
-      <div class="col-12">
-        <%= f.label :teacher_id,
-                    t('basics.teacher'),
-                    class: "form-label" %>
-        <%= helpdesk(t('admin.lecture.info.teacher'), false) %>
-        <%= f.select :teacher_id,
-                     options_for_select([[current_user.info, current_user.id]],
-                                        current_user.id),
-                     {},
-                     { class: 'selectize',
-                       data: { ajax: true,
-                               model: 'user',
-                               filled: false,
-                               placeholder: t('basics.enter_two_letters'),
-                               no_results: t('basics.no_results'),
-                               current: current_user.id,
-                               modal: modal } } %>
+      <div class="col-12 mt-4">
+        <% if current_user.admin? %>        
+          <%= f.label :teacher_id,
+                      t('basics.teacher'),
+                      class: "form-label" %>
+          <%= helpdesk(t('admin.lecture.info.teacher'), false) %>
+          <%= f.select :teacher_id,
+                       options_for_select([[current_user.info, current_user.id]],
+                                          current_user.id),
+                       {},
+                       { class: 'selectize',
+                         data: { ajax: true,
+                                 model: 'user',
+                                 filled: false,
+                                 placeholder: t('basics.enter_two_letters'),
+                                 no_results: t('basics.no_results'),
+                                 current: current_user.id,
+                                 modal: modal } } %>
+        <% else %>
```

Comment:
If this is from now on only accessible by admins: what is the `current_user` information used for in the form and why is it sent as data to the backend? Is it relevant?

Reply:
Hm, it seems that it is used in `app/views/lectures/new.coffee` where we have these lines

```coffee
# make sure that there will always be a teacher selected
teacherSelector = document.getElementById('lecture_teacher_id')
sel = teacherSelector.tomselect
sel.on 'blur', ->
  value = sel.getValue()
  if value == ''
    sel.setValue(teacherSelector.dataset.current)
  return
```

Do we still need this?